gloo/transport/uv/libuv.h

```diff
@@ -32,6 +32,10 @@
 #include <utility>
 #include <vector>
 
+#ifdef _WIN32
```

Comment:
If it's a standard library header, just include it unconditionally. No reason to guard it.